_health-care/_js/components/form-elements/ErrorableSelect.jsx

```diff
@@ -63,7 +63,7 @@ class ErrorableSelect extends React.Component {
     return (
       <div className={this.props.errorMessage ? 'usa-input-error' : undefined}>
         <label
-            className={this.props.errorMessage ? 'usa-input-error-label' : undefined}
+            className={this.props.errorMessage !== undefined ? 'usa-input-error-label' : undefined}
             htmlFor={this.selectId}>
```

Comment:
Will this make the field look wrong until the user types something in because it will be `undefined` on initialization correct?